api/api-doc/task_manager.json

```diff
@@ -319,6 +319,18 @@
             "sequence_number":{
                "type":"long",
                "description":"The running sequence number of the task"
+            },
+            "shard":{
+               "type":"long",
+               "description":"The number of a shard the task is running on"
+            },
+            "start_time":{
+               "type":"datetime",
+               "description":"The start time of the task"
+            },
+            "end_time":{
+               "type":"datetime",
+               "description":"The end time of the task (unspecified when the task is not completed)"
```

Comment:
How does an unspecified value look like? Is the field missing from the output altogether? Or does it have `null` or some other specific sentinel value?